03_launch_mgmt_cluster.sh

```diff
@@ -10,22 +10,36 @@ M3PATH="${GOPATH}/src/github.com/metal3-io"
 BMOPATH="${M3PATH}/baremetal-operator"
 CAPBMPATH="${M3PATH}/cluster-api-provider-baremetal"
 
+BMOREPO="${BMOREPO:-https://github.com/metal3-io/baremetal-operator.git}"
+BMOBRANCH="${BMOBRANCH:-master}"
+CAPBMREPO="${CAPBMREPO:-https://github.com/metal3-io/cluster-api-provider-baremetal.git}"
+CAPBMBRANCH="${CAPBMBRANCH:-master}"
+FORCE_REPO_UPDATE="${FORCE_REPO_UPDATE:-false}"
```

Comment:
Can you document these in `config_example.sh` ?

Reply:
ok, done